CMakeLists.txt

```diff
@@ -5,517 +5,604 @@ set(LIBIIO_VERSION_MAJOR 0)
 set(LIBIIO_VERSION_MINOR 19)
 set(VERSION "${LIBIIO_VERSION_MAJOR}.${LIBIIO_VERSION_MINOR}")
 if (WIN32)
-	string(TIMESTAMP BUILD_YEAR "%Y")
-endif()
+    string(TIMESTAMP BUILD_YEAR "%Y")
+endif ()
```

Comment:
the more popular version for the CMake constructs [at least in the CMake files of libiio] seems to be without the space;
i.e. `endif()`
can we make that the rule?
or, is there a convention listed somewhere?

Reply:
I think the CMake formatter will do that (no space). I will update.